hw_09/hw_09/CRUD.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace hw_09 {
+    public static class CRUD {
```

Comment:
- is not a good name for this class
- all static members inside CRUD type - is not good